### PR TITLE
Adds Teams API documentation

### PR DESCRIPTION
**Pull request by akavi**
WISOTT

On Staging: http://pd-developer-staging.herokuapp.com/documentation/rest/teams

### CODE REVIEW

**Comment by afolson** [comment]
It looks like all of the curl commands and JSON examples aren't rendering properly across all of these pages. As an example: http://pd-developer-staging.herokuapp.com/documentation/rest/teams/list

The problem should be solved with a new line after you open the code blocks/before the sample code. Other than that, it looks good to me.

**Comment by steverice** [comment]
None of your response examples are proper JSON. Needs more quotes.

File: source/documentation/rest/teams/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                <td>
+                    <a href="/documentation/rest/teams/update">PUT teams/:id</a>
+                </td>
+                <td>Update a team.</td>
+            </tr>
+            <tr>
+                <td>
+                    <a href="/documentation/rest/teams/delete">DELETE teams/:id</a>
+                </td>
+                <td>Delete ateam.</td>
```
Comment (kenrose) [inline source/documentation/rest/teams/index.html]: Typo.

File: source/documentation/rest/teams/index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                <td>
+                    <a href="/documentation/rest/teams/create">POST teams</a>
+                </td>
+                <td>Create a new team.</td>
+            </tr>
+            <tr>
+                <td>
+                    <a href="/documentation/rest/teams/show">GET teams/:id</a>
+                </td>
+                <td>Get details about a team.</td>
```
Comment (kenrose) [inline source/documentation/rest/teams/index.html]: Show detailed information about a team

File: source/documentation/rest/teams/index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+permalink: /documentation/rest/teams/
+---
+<ul class="breadcrumb">
+    <li>
+        <a href="/documentation/rest">REST API</a>
+    </li>
+    <span class="divider">/</span>
+    <li>Teams</li>
+</ul><div class="section">
+    <title></title>
```
Comment (steverice) [inline source/documentation/rest/teams/index.html]: What's this doing here?

File: source/documentation/rest/teams/index.html (hunk 2 above)
Comment (steverice) [inline source/documentation/rest/teams/index.html]: Whatever the copy, it should probably match [what's on the page itself](https://github.com/PagerDuty/devdocs/pull/5/files#diff-7db07205a012abd3bf016563db9a4213R18).

File: source/documentation/rest/teams/show.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    <p class="content-type-warning clearfix">Note: <code>Content-type</code> of the request <strong>must</strong> be <code>application/json</code></p>
+    <div class="clearfix"></div>
+    <h4>Response <span class="response-status">Status: 200 OK <a href="/documentation/rest/errors#http_codes"><i class="icon-question-sign"></i></a></span></h4>
+    <div class="response">
+        <pre>
+            <code class="language-javascript prettyprint linenums">{
+  team: {
+    id: PBAZLIU,
+    name: A Mob of Emus,
+    description: null
```
Comment (steverice) [inline source/documentation/rest/teams/show.html]: Mobs of Emus need no explanation.

File: source/documentation/rest/teams/index.html (hunk 3 above)
Comment (akavi) [inline source/documentation/rest/teams/index.html]: Ehhhhh, all this html was autogenned. Probably a wonky `if` somewhere.

**Pull request merged**